### ISSUE TITLE
cm.get_cmap doesn't work

### ISSUE THREAD

**Issue by csherwood-usgs** [SatelliteShorelines/CoastSeg#267]
My first try at mapping shorelines resulted in this error:

![image](https://github.com/SatelliteShorelines/CoastSeg/assets/1618995/aba7860d-de13-4684-97fc-8d7c87c7e38b)

Maybe the problem is related to this (but while it says "deprecated", I think it is still there...and, in fact, the code in cm.py in that folder has a `get_cmap` function).

So, if this were my code, I would try that workaround, but I don't know how to safely mess with your code. Do I make a local branch and do it there?

![image](https://github.com/SatelliteShorelines/CoastSeg/assets/1618995/de2ac5f5-14d2-490e-92dc-7394aaa4e403)

**Comment by 2320sharon** [comment]
Hi @csherwood-usgs 
Thank you for reporting this error. It looks like the version of matplotlib your environment installed has deprecated the colormap function  that the underlying coastsat code uses. You can safely create a branch and modify your code or just modify the code you have locally. You're perfectly fine to modify your code so long as you don't push it to the master branch which you don't have permissions for so you're safe changing your own copy.

I'll see if I can update the underlying coastsat code to fix this issue. It should be a pretty easy fix.

**Comment by 2320sharon** [comment]
@csherwood-usgs 
Good news! This issue is fixed in coastsat_package 0.1.73.
You can install it in your coastseg environment using the command below
```
conda activate coastseg
pip install coastsat-package==0.1.73
```

I'm also going to raise an issue in CoastSat so no one else encounters this issue

**Comment by 2320sharon** [comment]
This issue has been fixed and resolved in coastseg 1.2.11
